dev-client/src/store/persistence.ts

```diff
@@ -38,3 +39,9 @@ export const loadPersistedReduxState = () => {
     );
   }
 };
+
+export const patchPersistedReduxState = (state?: Partial<AppState>) => {
+  if (state) {
+    merge(state, {soilId: {soilChanges: {}}});
```

Comment:
@shrouxm suggested pulling in the initial soil state here -- I agree that this is the better solution, however it is not possible to do this at the moment because of dependency issues in the soil slice. We should create a tech-debt ticket to design a system to handle this more generally for all redux slices and solve the underlying dependency problems.